sdk/ai/azure-ai-projects/samples/evaluation/sample_agent_evaluations.py

```diff
@@ -43,7 +43,7 @@
 
 with DefaultAzureCredential(exclude_interactive_browser_credential=False) as credential:
 
-    with AIProjectClient(endpoint=endpoint, credential=credential) as project_client:
+    with AIProjectClient(endpoint=endpoint, credential=credential, api_version="latest") as project_client:
```

Comment:
api_version="lates" is temporary right? removed before //build? Perhaps put a comment to indicate this.

Reply:
Added a comment for it.